skxray/cdi.py

```diff
@@ -48,48 +48,31 @@
 logger = logging.getLogger(__name__)
 
 
-def squared_dist_2D(dims):
+def _dist(dims):
     """
-    Create array with pixel value equals squared euclidian distance
-    from array center in 2D.
+    Create array with pixel value equals to the distance from array center.
 
     Parameters
     ----------
     dims : list or tuple
-        shape of the data
+        shape of array to create
 
     Returns
     -------
-    array :
-        2D array to meet requirement
-    """
-    x_sq = (np.arange(dims[0]) - dims[0]/2)**2
-    y_sq = (np.arange(dims[1]) - dims[1]/2)**2
-    return x_sq.reshape([dims[0], 1]) + y_sq
-
-
-def dist(dims):
+    arr : np.ndarray
+        ND array whose pixels are equal to the distance from the center
+        of the array of shape `dims`
     """
-    Create array with pixel value equals euclidian distance from array center.
-
-    Parameters
-    ----------
-    dims : list or tuple
-        shape of the data
+    dist_sum = []
+    shape = np.ones(len(dims))
+    for idx, d in enumerate(dims):
+        vec = (np.arange(d) - (d-1)/2) ** 2
```

Comment:
if d is odd. 
```np.arange(d) - (d-1)/2``` and ```np.arange(d) - (d)/2``` are the same
if d is even, say 10, we basically get 
```
In [37]: np.arange(d) - (d-1)/2
Out[37]: array([-4, -3, -2, -1,  0,  1,  2,  3,  4,  5])

In [38]: np.arange(d) - (d)/2
Out[38]: array([-5, -4, -3, -2, -1,  0,  1,  2,  3,  4])
```
I will use d instead of d-1 to make the code consistent with other parts of the code I am going to send through PR.